src/databricks/labs/ucx/workspace_access/groups.py

```diff
@@ -90,6 +91,8 @@ def _get_group(self, group_name, level: GroupLevel) -> iam.Group | None:
             if group.display_name == group_name:
                 return group
 
+    @retried(on=[IOError])
```

Comment:
shouldn't you retry on `DatabrocksError` instead? what exception message were you getting to add retries, btw? we'll probably be getting more concrete exceptions in SDK soon, so i'd like comments with messages :) 

see https://github.com/databricks/databricks-sdk-py/pull/376

Reply:
I agree it would be better to use the specific `DatabrocksError `. Let me update this.

Reply:
I double-checked and indeed it was `DatabrocksError `. Code was updated accordingly.

Reply:
fixed now